libnavigation-base/src/main/java/com/mapbox/navigation/base/utils/DecodeUtils.kt

```diff
@@ -133,6 +142,12 @@ object DecodeUtils {
         }
     }
 
+    private fun LruCache<Pair<String, Int>, List<Point>>.clear() {
+        synchronized(lock = this) {
```

Comment:
I think synchronization here is not necessary, because `LruCache` is thread-safe.